### ISSUE TITLE
sort out accessibility of the `FilterableFacet` component

### ISSUE THREAD

**Issue by alicewriteswrongs** [mitodl/ocw-hugo-themes#383]

**Comment by alicewriteswrongs** [comment]
I'm un-assigning myself, but I did start some exploratory work on this on this branch: https://github.com/mitodl/ocw-hugo-themes/compare/ap/topic-widget so that could form the basis for resolving these issues. The main concerns are:

- tab navigation between large elements on the page and then arrow keys + spacebar to interact with them
- sorting out proper screenreader announce tags
- make sure that we use semantic HTML tags everywhere we should, i.e. `<button>` and _no_ `<div onclick=` fake buttons

there are also probably some visual design accessibility things we should be concerned with here too, contrast and so on.
